devito/finite_differences/differentiable.py

```diff
@@ -240,16 +240,17 @@ def laplace(self):
         derivs = tuple('d%s2' % d.name for d in space_dims)
         return Add(*[getattr(self, d) for d in derivs])
 
-    @property
-    def div(self):
+    def div(self, shift=None):
```

Comment:
turning this property into a function -- should we consider it a major API change?

Reply:
No this is not user facing, `div(f)` is and hasn't changed.